src/TestFramework/AbstractTestFrameworkAdapter.php

```diff
@@ -97,21 +97,21 @@ public function getInitialTestRunCommandLine(
     /**
      * Returns array of arguments to pass them into the Mutant Process
      *
-     * @param TestLocation[] $tests
+     * @param TestLocation[] $coverageTests
      *
      * @return string[]
      */
     public function getMutantCommandLine(
-        array $tests,
-        string $mutantFilePath,
+        array $coverageTests,
```

Comment:
psalm on level 7 generates an error:
```
ERROR: ParamNameMismatch - src/TestFramework/AbstractTestFrameworkAdapter.php:105:15 - Argument 1 of Infection\TestFramework\AbstractTestFrameworkAdapter::getMutantCommandLine has wrong name $tests, expecting $coverageTests as defined by Infection\AbstractTestFramework\TestFrameworkAdapter::getMutantCommandLine (see https://psalm.dev/230)
        array $tests,
```

that is why property has been renamed. (the next renamed for the same reason)